website/docs/sql-reference/statements/sql-select.md

```diff
@@ -0,0 +1,49 @@
+---
+id: select
```

Comment:
FYI not seeing statements appear in the sidebar

Reply:
Good catch, thanks!